### PR TITLE
chnages html sementic tag in headerlist

### LINKED ISSUE #69: headerList html semantic
Should be 

```
<aside> 
  <ul>
    <li>
```

### PR DESCRIPTION
**Pull request by AjRadwan**
**Changes div to** 

_aside > ul >  li_ 

This fixes #69

### CODE REVIEW

File: src/components/headersList/headersList.tsx [hunk 1]
```diff
@@ -29,23 +29,23 @@ function HeadersList({
 	}, headings);
 	return (
 		<aside>
-			<div>
+			<ul>
 				{data.map((i) => (
 					<div className="my-1">
```
Comment (pmvald) [inline src/components/headersList/headersList.tsx:34]: replace this with `<li className="my-1">`

File: src/components/headersList/headersList.tsx [hunk 2]
```diff
@@ -29,23 +29,23 @@ function HeadersList({
...
-						<a href={"#" + paramCase(i.value)} >
+						<li><a href={"#" + paramCase(i.value)} >
```
Comment (pmvald) [inline src/components/headersList/headersList.tsx:35]: see above

File: src/components/headersList/headersList.tsx [hunk 3]
```diff
@@ -29,23 +29,23 @@ function HeadersList({
...
 							<span>{i.value}</span>
-						</a>
+						</a></li>
 						{i.childs.length > 0 &&
 							i.childs.map((j) => (
 								<div className="ms-h my-h">
```
Comment (pmvald) [inline src/components/headersList/headersList.tsx:40]: same as above, replace this div with li

**Review by pmvald — CHANGES_REQUESTED** [review]

File: src/components/headersList/headersList.tsx (hunk 3 above)
Comment (AjRadwan) [reply src/components/headersList/headersList.tsx]: I changed it according to what you told me.  I hope this works fine now

**Review by pmvald — APPROVED** [review]

**Pull request merged**